From d50f02b661698243fbeb556bb407c13da9ae4718 Mon Sep 17 00:00:00 2001
From: Friedemann Kleint <Friedemann.Kleint@qt.io>
Date: Tue, 4 Dec 2018 09:25:29 +0100
Subject: [PATCH] shiboken: Fix build with Qt 5.13

Add missing include, fixing:

sources/shiboken2/ApiExtractor/clangparser/clangutils.h:91:17: error: field 'childMessages' has incomplete type 'QStringList'

appearing in Qt 5.13 due to some header rearrangements.

Change-Id: Iab22538af8688ebb3dcbaf0e1042b1198bde5800
Reviewed-by: Cristian Maureira-Fredes <cristian.maureira-fredes@qt.io>
---
 sources/shiboken2/ApiExtractor/clangparser/clangutils.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sources/shiboken2/ApiExtractor/clangparser/clangutils.h b/sources/shiboken2/ApiExtractor/clangparser/clangutils.h
index b290aac9..db2db626 100644
--- a/sources/shiboken2/ApiExtractor/clangparser/clangutils.h
+++ b/sources/shiboken2/ApiExtractor/clangparser/clangutils.h
@@ -32,6 +32,7 @@
 #include <clang-c/Index.h>
 #include <QtCore/QPair>
 #include <QtCore/QString>
+#include <QtCore/QStringList>
 #include <QtCore/QVector>
 
 #include <functional>
-- 
2.23.0

